indico/modules/rb/forms/settings.py

```diff
@@ -42,6 +42,17 @@ class SettingsForm(IndicoForm):
     notification_before_days_monthly = IntegerField(_('Send booking reminders X days before (monthly)'),
                                                     [InputRequired(), NumberRange(min=1, max=30)])
     notifications_enabled = BooleanField(_('Reminders enabled'))
+    notification_before_end_daily = IntegerField(_('Send reminders about ending bookings X days before their end date '
+                                                   '(daily)'),
+                                                 [InputRequired(), NumberRange(min=1, max=30)])
+    notification_before_end_weekly = IntegerField(_('Send reminders about ending bookings X days before their end date '
+                                                    '(weekly)'),
+                                                  [InputRequired(), NumberRange(min=1, max=30)])
+    notification_before_end_monthly = IntegerField(_('Send reminders about ending bookings X days before their end '
+                                                     'date (monthly)'),
+                                                   [InputRequired(), NumberRange(min=1, max=30)])
+    notifications_before_end_enabled = BooleanField(_('Reminders of ending bookings enabled'))
```

Comment:
I think this text doesn't read very well. It's already confusing what this field does. What about "Send reminders when bookings are about to end" or similar?